instrumentation/opentelemetry-instrumentation-pyramid/src/opentelemetry/instrumentation/pyramid/__init__.py

```diff
@@ -94,9 +92,12 @@
     trace_tween_factory,
```

Comment:
could you not add a pylint ignore here instead of adding the pointless statement below?

Reply:
Sorry what do you mean? Is there a pylint ignore statement? :thinking: I was under the impression that it was only possible to disable certain specific errors, with `pylint: disable=specific-error`. Do you mean to use `pylint: disable=unused-import`?

If I do this:

```
from opentelemetry.instrumentation.pyramid.callbacks import (
    ...
    trace_tween_factory,  # pylint: disable:unused-import
)
```

And then I don't add `trace_tween_factory` below, then `flake8` will fail because we have an unused import and probably `pylint` would have failed afterwards because we have an useless `disable=...` comment (remember that `pylint` never complained about this symbol being imported and unused.

Reply:
you can ignore both in the same line:

`# noqa # pylint: disable=unused-import`

Reply:
Ok, let me try that...

Reply:
Ok, no, this just makes `isort` angry. Can we merge as it is and then address the unused symbol in another PR?

Reply:
👍 opened https://github.com/open-telemetry/opentelemetry-python-contrib/issues/648 to track this